python/lib/dcoscli/tests/integrations/test_ssl.py

```diff
@@ -76,5 +74,5 @@ def _ssl_error_msg():
         "run: 'dcos config set core.ssl_verify <value>'\n"
         "<value>: Whether to verify SSL certs for HTTPS or path to certs. "
         "Valid values are a path to a CA_BUNDLE, "
-        "True (will then use CA certificates from certifi), "
+        "True (will then use system CA certificates), "
```

Comment:
Wait, are we testing `dcos/dcos-cli` code here? This is messed up. Anyway, it shouldn't block this PR.

Reply:
Yes we are :( Unfortunately this is the only test to check `ssl_verify` and it uses marathon command.

Reply:
OK, it uses marathon command because commands in CLI are not using TLS.